pages/getting-started/developers.md

```diff
@@ -140,6 +140,24 @@ You can also email us directly at [uswebdesignstandards@gsa.gov](mailto:uswebdes
 **For more information, visit:
 [https://pages.18f.gov/frontend/css-coding-styleguide/](https://pages.18f.gov/frontend/css-coding-styleguide/)**
 
+## JS customization
+
+The JavaScript for the standards is separated into components in the same manner as the visual interface which is all initialized with event handlers when the DOM is ready. These components are accessible as CommonJS modules that can be required in other JavaScript files which then must be fuilt for the browser. The components are currently not accessible in the global browser scope, but can be easily extended to be included by requiring `components` and setting it to a global scope:
+
+```js
+window.uswds = require('./components');
+```
+
+Each component has a standardized interface that can be used to extend it further. The primary methods each component has are as follows:
+
+- `on`: Initialize a component's JavaScript behavior by passing the root element, such as `window.document`.
```

Comment:
It might be worth mentioning something about the fact that we're using a delegation model here, for folks who are unfamiliar with it?  Otherwise readers might get confused, e.g. wondering why anyone would want to turn `window.document` into an accordion...